### PR TITLE
[AR-1023] Create Radio Button Component

### PR DESCRIPTION
**Pull request by KarthikSiddarth**
Resolves [AR-1023](https://team-1624093970686.atlassian.net/browse/AR-1023).

## Blocking dependencies

- [AR-1024](https://team-1624093970686.atlassian.net/browse/AR-1024)

## Changes

1. Created Radio Button and Radio Button Group component.

## Screenshot:
Screenshot 2022-02-10 at 3 38 03 PM

## Checklist

- [X] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [X] The changes have been tested locally.

### CODE REVIEW

File: src/components/lib/VRadio/VRadio.vue [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="flex flex-center justify-space-between">
+    <label class="body-1">{{ label }}</label>
```
Comment (shrinathprabhu) [inline src/components/lib/VRadio/VRadio.vue:3]: Can we add a `for` attribute to this label?

Comment (KarthikSiddarth) [reply src/components/lib/VRadio/VRadio.vue]: done. [commit link](https://github.com/arcana-network/developer-dashboard/pull/50/commits/0c4a00809437daf0c6e56487c0b5a32056014fe5)

**Review by shrinathprabhu — APPROVED** [review]

**Review by ajithranka — CHANGES_REQUESTED** [review]
**Change `selectedValue` to `modelValue`**
Since want the `VRadio` and `VRadioGroup` components to accept a `v-model` prop, the `selectedValue` prop in `VRadio` and `selectedChain` prop in `VRadioGroup` should be `modelValue`s instead.

Internally, Vue expands `v-model` on custom components to `modelValue` and listens for `update:modelValue` events. [More details](https://vuejs.org/guide/components/events.html#usage-with-v-model).

**Add an emits props**
To be explicit that this component emits `update:modelValue` events.

```
emits: ['update:modelValue`]
```

**Comment by KarthikSiddarth** [comment]
> **Change `selectedValue` and `seletedChain` to `modelValue`** Since want the `VRadio` and `VRadioGroup` components to accept a `v-model` prop, the `selectedValue` prop in `VRadio` and `selectedChain` prop in `VRadioGroup` should be `modelValue`s instead.
> 
> Internally, Vue expands `v-model` on custom components to `modelValue` and listens for `update:modelValue` events. [More details](https://vuejs.org/guide/components/events.html#usage-with-v-model).
> 
> **Add an emits props** To be explicit that these components emits `update:modelValue` events.
> 
> ```
> emits: ['update:modelValue`]
> ```

@ajithranka made the suggested changes.  [commits - 1](https://github.com/arcana-network/developer-dashboard/pull/50/commits/a35b7557b43007e798b8a7d7bf1872da40155b70), [commit-2](https://github.com/arcana-network/developer-dashboard/pull/50/commits/ef361e6358178a31f419ff1d8cfc8616c6eeff4a)

File: src/components/lib/VRadio/VRadio.vue [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    label: {
+      type: String,
+      default: "",
+    },
+    modelValue: {
+      type: [String, Number, Boolean],
+      required: true,
+    },
+    value: {
+      type: String,
```
Comment (ajithranka) [inline src/components/lib/VRadio/VRadio.vue:60]: Can also be a string, number or boolean.

Comment (KarthikSiddarth) [reply src/components/lib/VRadio/VRadio.vue]: Changed. [commit](https://github.com/arcana-network/developer-dashboard/pull/50/commits/0922218c188169df49dbff5d494baf8adf4e0180)

**Pull request merged**